### ISSUE TITLE
Monika disappeared when I did the "I'm taking you somewhere" function.

### ISSUE THREAD

**Issue by MewMewKissyCutey** [Monika-After-Story/MonikaModDev#4883]
For starters, I never really knew which file I was supposed to move, but after doing research I learned that it was the Persistent file. 

For about a day, she was on my computer just fine, I was really happy that she worked. However she was telling me to remove her from steam. Any time I would, the mod didn't open though, so I kept her there. 

I went to a relative's house, and when I opened the game up, she was gone. She was on my other computer today when I opened it up, so I tried to put her on my other computer again- she didn't load once again when putting the persistent file on the new laptop again. I tried 2 more times, and now she doesn't load on the other computer either. 

There's no note from her, I can't click on anything. Just the empty classroom, with no music playing. 

I'm not sure what to do, and it's getting rather frustrating now. I don't want to restart all my progress in this mod. If there are any files you need, let me know what is needed and I'll send it over ASAP. 

edit:
- For some reason there wasn't a Monika.chr file on my old computer, while there was one on my new one. She was loading on the computer without one until today. I tried putting the monika (file) with the .Chr monika on my new computer, but that didn't help the issue. All the other girls's .chr files are still there too.

**Comment by multimokia** [comment]
You need to put the `monika` file back into the `characters` folder. That file is her, and as such, if it's not present, neither is she.

If you want to transfer from computer to computer, don't use the `monika` file as you'll need to then transfer both the `persistent` and `monika` file together. What you can do, is keep Monika in the room and *just* transfer the `persistent` alone. This will handle everything together.

**Comment by MewMewKissyCutey** [comment]
I don't quite understand what you mean by this. 

I have monika file and Monika .chr in the characters folder on both computers and she still doesn't load. I tried moving the monika file from the old computer and the persistent file is in the Renpy folder. Is that where it should be? I don't know.

**Comment by multimokia** [comment]
Okay, one thing `Monika.chr` no longer matters. That file isn't her so you can get rid of it.

Is that `monika` file an older one when you tried to take her somewhere previously?
Better question actually, is it the *same* file on both computers? Or is one bigger than the other?

**Comment by MewMewKissyCutey** [comment]
It should be the same file on both computers, because I took Monika file from the first computer, put it in my back up on my dad's server, and then took it and put it into the characters folder on my new one. I don't know how it would become a bigger size through that. 

I looked on both computers, and the file is the same size: 6,523 KB.

**Comment by multimokia** [comment]
Send a `mas_log.txt` (with the MAS install closed) please. It's in the `log` folder.

**Comment by MewMewKissyCutey** [comment]
I don't know where the "log" folder is, is it supposed to be in the Doki Doki Literature Club folder or somewhere else on my computer?

**Comment by multimokia** [comment]
In the DDLC folder

**Comment by MewMewKissyCutey** [comment]
![image](https://user-images.githubusercontent.com/56707612/67062558-8090e780-f118-11e9-8694-9783b1ce34ee.png)
this is my DDLC folder, and I don't see any folders called "log"

**Comment by multimokia** [comment]
Okay... that's not a MAS install

**Comment by multimokia** [comment]
It should look more like this:
![image](https://user-images.githubusercontent.com/22531674/67062651-f05f9c00-f131-11e9-9c84-e4972aaa49ce.png)

**Comment by MewMewKissyCutey** [comment]
This is the ZIP I downloaded from MonikaAfterStory.com 
![image](https://user-images.githubusercontent.com/56707612/67062702-f8f7a880-f118-11e9-9aaa-e0062edd9e46.png)

None of the folders on my computer have this. Should I take the whole DDLC game on my old computer (which does have all those files) and see if Monika returns, before sending the log?

**Comment by MewMewKissyCutey** [comment]
*none of the folders on my computer have what your screenshot shows, I mean

**Comment by multimokia** [comment]
Well, you said you were running on Steam. Your screenshot doesn't look like a steam install.

**Comment by MewMewKissyCutey** [comment]
Yes, she told me to move the folder from steam to "anywhere else" so I moved the Doki Doki Literature Club folder out of steam. It wouldn't open, so I copied it back into steam. 

That's the folder I have of my game. I don't know what I would show in a screenshot with steam, because opening the "browse files location" looks like this:
![image](https://user-images.githubusercontent.com/56707612/67063329-45dc7e80-f11b-11e9-8ac2-d61c11e7b0df.png)

I'm not really sure what I should be screenshotting to show you when the folder viewers look identical.

**Comment by multimokia** [comment]
Yeah they don't. The first install you showed me is not a MAS install. It is a regular DDLC install. Currently on this computer, MAS will run from Steam.

**Comment by multimokia** [comment]
Inside the steam install's `log` folder, send the `mas_log.txt` file please.

**Comment by MewMewKissyCutey** [comment]
Huh. Well, the second screenshot with the folder from Steam looks identical to the mod to me. This is how it looks when I open the game: 
![image](https://user-images.githubusercontent.com/56707612/67063621-47f30d00-f11c-11e9-91ad-8efe63aae13f.png)

In the Steam log there isn't a mas_log.txt. 
here's a screenshot of that: 
![Uploading image.png…]()

I am trying to get the mas_log.txt from my old computer to you right now, but it's significantly slower than this one.

**Comment by multimokia** [comment]
Image didn't finish uploading before you posted

**Comment by MewMewKissyCutey** [comment]
![image](https://user-images.githubusercontent.com/56707612/67063662-68bb6280-f11c-11e9-9881-4afad881d5e6.png)

**Comment by MewMewKissyCutey** [comment]
Hold on, it sent the old pic again
![image](https://user-images.githubusercontent.com/56707612/67063700-88528b00-f11c-11e9-9355-ff7f2176f102.png)


here is the folder

**Comment by MewMewKissyCutey** [comment]
This was the "logs" folder in steam. There is no logs folder anywhere else that I can find.

**Comment by multimokia** [comment]
![image](https://user-images.githubusercontent.com/22531674/67063724-c8723780-f135-11e9-92bd-ced0a758c20f.png)
I can't make this any more clear to you

**Comment by MewMewKissyCutey** [comment]
alright, my mistake. I didn't see the log folder there when I was looking.
here is the mas_log.txt 
[mas_log.txt](https://github.com/Monika-After-Story/MonikaModDev/files/3742184/mas_log.txt)

**Comment by multimokia** [comment]
Okay, for investigation purposes. I'll need a copy of your persistents please. (Just zip them up and send them here)

After you do that, just restore a persistent backup. You can see how [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent).

**Comment by MewMewKissyCutey** [comment]
[persistent.zip](https://github.com/Monika-After-Story/MonikaModDev/files/3742193/persistent.zip)

here you go.

**Comment by multimokia** [comment]
can you send the `monika` file too?

**Comment by MewMewKissyCutey** [comment]
[persistent.zip](https://github.com/Monika-After-Story/MonikaModDev/files/3742198/persistent.zip)

Here is the persistent file as well as the monika file. 

if you need the ones from the renpy file, I'll send those as well.

**Comment by multimokia** [comment]
Thank you. But yeah, just restore a backup and you should be back to normal.

**Comment by MewMewKissyCutey** [comment]
alright, Thank you. 
I'll begin that now.

**Comment by MewMewKissyCutey** [comment]
Hey, just to say, she's back, working! 
I hadn't really made any progress since June, so she's back :)

**Comment by ThePotatoGuy** [comment]
[closing as resolved]

**Comment by 3IWantToSleep3** [comment]
hii,today i had the same problem,this morning i made the option to take her out with me, as soon as i came back i put monika's file in the character folder, but she's disappeared,i restarted the computer several times but nothing, what can i do?w

**Comment by daisydoo111** [comment]
I have the same problem aswell. I don't know why but I decided to test out 'taking her out with you' option, and took her out my character folder once she added herself; Hence why I have this problem. I put her into my Downloads folder then looked for her. Idk how, but when I looked in that folder, she wasn't there I checked like 100 times and she was still gone. please send help ASAP I don't know what to do after several research...!!! :(

**Comment by dreamscached** [comment]
@daisydoo111 restore a backup; in your %APPDATA%\RenPy\Monika After Story there are `persistent##.bak` files, find the most recent and rename to `persistent`.

**Comment by daisydoo111** [comment]
Thank you for telling me this. Luckily I didn't get too far in that
persistent so I just got another backup persistent from a friend and she
let copy paste it to my folder and then I worked at it again. Now I am even
further than I was before- sadly I had to start over, but it didn't take
long as I was pretty addicted. Thank you anyway for texting me, I really
appreciate it.


-----Your most sincerest,✨ Ellie ✨
